src/client.rs

```diff
@@ -508,30 +512,51 @@ impl<'mpesa, Env: ApiEnvironment> Mpesa<Env> {
     ///
     /// # Errors
     /// Returns `EncryptionError` variant of `MpesaError`
+
     pub(crate) fn gen_security_credentials(&self) -> MpesaResult<String> {
```

Comment:
Suggestion, perhaps we could keep the original implementation under `#[cfg(not(target_arch = "wasm32"))]` and the wasm implementation under `#[cfg(target_arch = "wasm32")]`